process/css-styleguide/spacing.css

```diff
@@ -0,0 +1,346 @@
+.p-0 {
+  padding: 0px;
+}
+
+.p-1 {
+  padding: 0.25rem;
+}
```

Comment:
I think that you can make variables with all these values (`0.25rem`, `0.5rem`, ...). This way, if you want to update one of them someday, you only need to update it in one place and not everywhere.